website/src/ts/components/install-info.ts

```diff
@@ -0,0 +1,34 @@
+import { LitElement, type TemplateResult, css, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+@customElement('ds-install-info')
+export class InstallInfo extends LitElement {
+  @property() package?: string;
+
+  override render(): TemplateResult {
+    return html`
+      <section>
+        <h2>Installation</h2>
```

Comment:
Installation should contain a link like other headings, right now it's not rendered in the navigation on the right:

<img width="1158" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/0e0bebff-9011-4422-b9d4-e44b606d3e94">